metricbeat/module/system/filesystem/helper.go

```diff
@@ -111,6 +111,8 @@ func filterFileSystemList(fsList []sigar.FileSystem) []sigar.FileSystem {
 
 // GetFileSystemStat retreves stats for a single filesystem
 func GetFileSystemStat(fs sigar.FileSystem) (sigar.FileSystemUsage, error) {
+	// Sigar, in line with the underlying `statfs` call on unix, refers to inodes as `files` in the fields.
+	// There's no performant, filesystem-independent way to get *just* the count of files, so for most uses inodes will be close enough for jisting.
```

Comment:
```suggestion
	// There's no performant, filesystem-independent way to get *just* the count of files, so for most uses inodes will be close enough.
```

Is jisting a word? I don't know, but the sentence works fine without it.

Thanks for the comments!

Reply:
I was thinking of "gist" and somehow ended up there. I think I need more caffeine...